h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -2987,3 +2987,20 @@ def check_ignore_cols_automl(models,names,x,y):
         else:
             assert set(h2o.get_model(model).params["ignored_columns"]["actual"]) == set(names) - {y} - set(x), \
                 "ignored columns are not honored for model " + model
+
+def check_sorted_2_columns(frame1, sorted_column_indices, prob=0.5):
```

Comment:
What is the reason to only check some randomly picked cells of the data frame?

Reply:
To speed up the test.